base/cvd/cuttlefish/host/commands/assemble_cvd/boot_image_utils.cc

```diff
@@ -179,8 +179,7 @@ void UnpackRamdisk(const std::string& original_ramdisk_path,
   int cpio_status;
   do {
     LOG(ERROR) << "Running";
-    cpio_status = Command(HostBinaryPath("toybox"))
-                      .AddParameter("cpio")
+    cpio_status = Command(CpioBinary())
```

Comment:
Will this block enabling substitution for `assemble_cvd` until `cpio` is available internally?

Can this get a symlink like ag/33707825 so that `cpio` will work out of `cvd-host_package.tar.gz` as well?